src/generated/_definitions.ts

```diff
@@ -33,13 +33,13 @@ export default [
     bgn_pcgn_alt,
     bs_2979,
     bs_2979_alt,
-    gost_779,
-    gost_779_alt,
-    gost_7034,
     gost_16876,
     gost_16876_alt,
     gost_52290,
     gost_52535,
+    gost_7034,
+    gost_779,
+    gost_779_alt,
```

Comment:
The order does not matter here, does it?

Reply:
Absolutely